src/services/orders.js

```diff
@@ -0,0 +1,154 @@
+const { Orders, Product } = require('../models');
+const { OrderStatusesService } = require('../services');
+
+const orderStatusesService = new OrderStatusesService();
+
+class OrdersService {
+  async createOrder (req, res) {
+    const orderStatus = await orderStatusesService.getStatusInProgress();
+    const order = new Orders(req.body);
+    order.status = orderStatus._id;
+    await order.save();
+    return res.json(true);
+  }
+
+  async getOrders (req, res) {
+    const query = req.query;
+    Object.keys(query).forEach((key) => {
+      if (!query[key]) {
+        delete query[key];
+      }
+    });
+
+    const count = Orders.find(query).countDocuments().exec();
+    const orders = await Orders.find(query)
+      .skip(+query.skip || 0)
+      .limit(+query.take || 50)
+      .exec();
+
+    return {
+      total_size: count,
+      list: orders ? res.json(orders) : res.json([])
+    };
+  }
+
+  async updateOrder (req, res) {
+    const order = await Orders.findOne({ _id: req.params.id }).exec();
```

Comment:
You can actually get only in progress order, since we cannot update resolved or rejected:
        const order = await Orders.findOne({_id: req.params.id, status: statuses.find(({name}) => name === 'In progress')._id}).exec();

        if (!order) {
            return res.boom.notFound('Order not found or it is already resolved or rejected');
        }